src/manifolds/Rotations.jl

```diff
@@ -330,7 +330,6 @@ function Random.rand!(
         (manifold_dimension(M) == 0) && return fill!(pX, 0)
         A = σ .* randn(rng, representation_size(M))
         pX .= triu(A, 1) .- transpose(triu(A, 1))
-        normalize!(pX)
```

Comment:
I would have red this line as we get back to a determinant 1 and for points this might be necessary? Yes I know you do tangents here, but maybe we should still normalise points ;) ?

Reply:
Normalization doesn't make determinant equal to 1 though. Random point generation seems to work correctly, and tangent vector generation doesn't need normalization.

Reply:
I do agree that tangent vectors do not need this :)
Then I maybe had the wrong thing in mind what is normalised here. If tests pass, this should be fine with me then.